Library/Homebrew/test/language/java_spec.rb

```diff
@@ -18,38 +18,38 @@
   end
 
   describe "::java_home" do
-    it "returns valid JAVA_HOME if version is specified" do
+    it "returns valid JAVA_HOME if version is specified", :needs_macos do
```

Comment:
This also requires having `openjdk` installed, right? Would be good to have a test for the Linux functionality too.

Reply:
Yes, I think it does. I was just duplicating the old test here. Unfortunately, it doesn't seem to work 😅

Reply:
Would need to add another stub to the `before do` to allow it to be an executable. That makes the test fairly pointless, though, if we're asserting on something we're stubbing to be the case. I'd suggest instead fixing up the equality value here and adding a `:needs_linux` equivalent test for the non-macOS functionality.

Reply:
Hmmm, ok, gotcha. Will try that.